test/src/e2e.long/staking.test.ts

```diff
@@ -0,0 +1,346 @@
+// Copyright 2018-2019 Kodebox, Inc.
+// This file is part of CodeChain.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import * as chai from "chai";
+import * as chaiAsPromised from "chai-as-promised";
+chai.use(chaiAsPromised);
+const expect = chai.expect;
+import { toHex } from "codechain-sdk/lib/utils";
+import { PlatformAddress, H256 } from "codechain-primitives/lib";
+import "mocha";
+import {
+    faucetAddress,
+    faucetSecret,
+    stakeActionHandlerId,
+    validator0Address,
+    validator0Secret,
+    validator1Address,
+    validator2Address,
+    validator3Address
+} from "../helper/constants";
+import { PromiseExpect } from "../helper/promise";
+import CodeChain from "../helper/spawn";
+
+const RLP = require("rlp");
+
+describe("Staking", function () {
```

Comment:
Is it just moving tests? or do you modify tests?

Reply:
Seperated commit. Refer to https://github.com/CodeChain-io/codechain/pull/1374